extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -588,6 +605,16 @@ export class SchemaCompareResult {
 		});
 	}
 
+	private toggleNoChanges(enable: boolean): void {
```

Comment:
nit: toggle usually implies that you switch the state to the opposite state. I'd suggest changing this to something like setButtonStates(enableButtons: boolean)

Reply:
changed to setButtonStatesForNoChanges(enableButtons: boolean)